packages/react/src/atoms/labels/base.tsx

```diff
@@ -13,6 +13,7 @@ export interface LabelBaseProps {
     color?: any;
     font?: any;
   };
+  onClick?: (e: any) => void;
```

Comment:
@Subbu-betalectic pls try to give a better type than `any`. If you hover over the onclick function in label base you will be able to find the type of event `e` for eg `React.MouseEvent<HTMLElement>`

Reply:
Pls make these suggested changes.